homeassistant/components/qvr_pro/__init__.py

```diff
@@ -30,7 +30,7 @@
                 vol.Required(CONF_HOST): cv.string,
                 vol.Required(CONF_USERNAME): cv.string,
                 vol.Required(CONF_PASSWORD): cv.string,
-                vol.Optional(CONF_PORT): cv.port,
+                vol.Optional(CONF_PORT, default=8080): cv.port,
```

Comment:
Please create a constant for this default value e.g. `DEFAULT_PORT`:

```suggestion
                vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
```